Makefile

```diff
@@ -0,0 +1,44 @@
+AUTOSCALER_SCHEDULER_IMG ?= kube-autoscale-scheduler:dev
+AUTOSCALER_AGENT_IMG ?= autoscaler-agent:dev
+VM_INFORMANT_IMG ?= vm-informant:dev
+EXAMPLE_VM_IMG ?= vm-example:dev
+
+.PHONY: docker-build
+docker-build:
+	docker buildx build \
+		--tag $(AUTOSCALER_SCHEDULER_IMG) \
+		--build-arg "GIT_INFO=n/a" \
+		--file build/autoscale-scheduler/Dockerfile \
+		.
+	docker buildx build \
+		--tag $(AUTOSCALER_AGENT_IMG) \
+		--build-arg "GIT_INFO=n/a" \
+		--file build/autoscaler-agent/Dockerfile \
+		.
+	docker buildx build  \
+		--tag $(VM_INFORMANT_IMG) \
+		--build-arg "GIT_INFO=n/a" \
+		--file build/vm-informant/Dockerfile \
+		.
```

Comment:
not a blocker, but it would be good to actually include the git info here. If that seems like a pain to sort out, I'm happy to take care of it after merging